packages/theme/src/themes.ts

```diff
@@ -56,6 +56,10 @@ export const lightTheme = createTheme({
     link: '#0A6EBE',
     gold: yellow.A700,
     sidebar: '#171717',
+    pinSidebarButton: {
```

Comment:
this is a bit too specific imo. Maybe we can try iconForeground + iconBackground, or `{ icon: { foreground, background }}`? Could even just put there colors in the component and switch on palette.type too tbh.